### PR TITLE
Remove unpopulated values

### PR DESCRIPTION
**Pull request by MightyAx**
Widen the definition of unpopulated values not to import into the JSON object.

### CODE REVIEW

File: excel/clean.py [hunk 1]
```diff
@@ -44,7 +44,8 @@ def entity_has_attribute(object_data: dict, attribute: str) -> bool:
 
 
 def value_populated(value: str) -> bool:
```
Comment (ke4) [inline excel/clean.py]: Could we rename it `is_value_populated`, please?
And use it accordingly.

Thanks

**Review by ke4 — CHANGES_REQUESTED** [review]
A small rename request.

Tx

File: excel/clean.py (hunk 1 above)
Comment (MightyAx) [reply excel/clean.py]: change made

**Review by ke4 — APPROVED** [review]
Perfect. Thanks for the changes. 💯  👍

**Pull request merged**